src/main/java/com/altenheim/kalender/controller/viewController/SearchViewController.java

```diff
@@ -59,30 +87,66 @@ else if (button.equals(btnBack) && userStep == 1)
         
         int currentIndex = userStep - 1;
         int requestedIndex = userStep - 1 + incrementor;
-        changeViewState(allSteps[currentIndex], allSteps[requestedIndex]); 
-        changeStepsButtonState(images[currentIndex], images[requestedIndex]);        
+        changeViewState(allSteps[currentIndex], allSteps[requestedIndex], images[currentIndex], images[requestedIndex]); 
         userStep += incrementor; 
         txtHeaderStep.setText(headings[currentIndex]);
     }
 
-    private void changeViewState(VBox deactivate, VBox activate)
+    private void changeViewState(VBox deactivate, VBox activate, Circle currentC, Circle nextC)
     {
         deactivate.setDisable(true);
         deactivate.setVisible(false);
         activate.setDisable(false);
         activate.setVisible(true);
+        currentC.setId("customCircleInactive");
+        nextC.setId("customCircleActive");   
     }
 
-    private void changeStepsButtonState(Circle currentC, Circle nextC)
+    @FXML
+    private void testUpdate(ActionEvent event) throws IOException, ClassNotFoundException
+    {   /*
+        var testEntry = entryFactory.createUserSettingsEntry(timeStart.getValue(), timeEnd.getValue());
+        var result = smartSearch.findAvailableTimeSlot(testEntry, (int)sliderAppointmentDuration.getValue());
+        for (var entry : result) 
+        {
+            SuggestionsModel.addToList(entry.getStartTime(), entry.getEndTime());
+            System.out.println(entry.getStartTime() + " " + entry.getEndTime());
+        }
+        iOController.writeCalendarFiles(null);
+
+        for (int i = 0; i < 5; i++)        
+            entryFactory.createRandomCalendarList("Test " + i);
+        iOController.writeCalendarFiles();*/
+        entryFactory.createContactsList(100);
+        iOController.saveContactsToFile();     
+    }
+
+    @FXML
+    private void resetTest(ActionEvent event) throws IOException, ClassNotFoundException
     {
-        currentC.setId("customCircleInactive");
-        nextC.setId("customCircleActive");        
+        //SuggestionsModel.data.clear();
+        //settings.setScrapingInterval(settings.getScrapingInterval()+1000);        
     }
 
+
     public void changeContentPosition() 
+    {        
+    }
+
+    private TableView<SuggestionsModel> createTable()
     {
-        
-    }      
-}
+        TableColumn<SuggestionsModel, String> startColumn = new TableColumn<>("Start");
+        TableColumn<SuggestionsModel, String> endColumn = new TableColumn<>("Ende");
+        TableView<SuggestionsModel> table = new TableView<SuggestionsModel>(SuggestionsModel.data);
 
+        startColumn.setCellValueFactory(new PropertyValueFactory<SuggestionsModel, String>("startTime"));
```

Comment:
Die Tables müssen wir leider wie es scheint dynamisch bauen, da hier OberservableCollections durchaus Sinn machen und diese mit durch FX initialisierte Objekte nicht so richtig zu funktionieren scheinen. Aber da Tables UI technisch gesehen trivial sind soltle das kein Prob sein. Aber wenn da jemand ne Idee hat oder das mal anders testen will könnte das als Verständnisgrundlage dienen (die Property Value Factory registriert die Veränderungen in der definierten Klasse (SuggestionsModel)).